syntax = "proto3";

message Path {
  string value = 1;
}

message Value {
  int32 value = 1;
}

message PathFiles {
  repeated string values = 2;
}

service FS {
  rpc ListFiles(Path) returns (PathFiles){};
  rpc OpenFile(Path) returns (Value){};
  rpc CloseFile(Path) returns (Value){};
}
